Detections/MultipleDataSources/SucessfullSiginFromPhingLink.yaml

```diff
@@ -0,0 +1,143 @@
+id: 779731f7-8ba0-4198-8524-5701b7defddc
+name: M365D Alerts Correlation to non microsoft Network device network activity involved in successful signing Activity
+description: |
+  'This content is employed to correlate with M365D phishing-related alerts. It focuses on instances where a user successfully connects to a phishing URL from a non-Microsoft network device and subsequently makes successful signing attempts from the phishing IP address.'
```

Comment:
It would be good to expand the M365D acronym here.

Reply:
signing attempts -> sign-in attempts

Reply:
Addressed